prime-dotnet-webapi/Configuration/OrganizationTypeConfiguration.cs

```diff
@@ -11,7 +11,7 @@ public override ICollection<OrganizationType> SeedData
             {
                 return new[] {
                     new OrganizationType { Code = 1, Name = "Health Authority", CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
-                    new OrganizationType { Code = 2, Name = "Community Practice", CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
+                    new OrganizationType { Code = 2, Name = "Private Community Health Practice", CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
```

Comment:
Is this supposed to change for both enrollees and site reg?

Reply:
Yah, it's a change across the board